GliaWidgets/View/Chat/Entry/Upload/FileUploadListView.swift

```diff
@@ -10,7 +11,13 @@ class FileUploadListView: UIView {
     private let stackView = UIStackView()
     private var heightLayoutConstraint: NSLayoutConstraint!
     private let style: FileUploadListStyle
-    private let kMaxUnscrollableViews = 3
+    private var maxUnscrollableViews: Int {
+        if UIApplication.shared.preferredContentSizeCategory <= .accessibilityMedium {
```

Comment:
Let's create managed dependency for UIApplication and inject it here via Environment. This dependency will be within UIKitBased namespace.

Reply:
I've created separate ticket [MOB-1345](https://glia.atlassian.net/browse/MOB-1345) for that fix. I'll fix it when @kuldar-daniel will merge his [PR](https://github.com/salemove/ios-sdk-widgets/pull/305)